style/icons/shield40_us_ky_parkway.svg

```diff
@@ -1,5 +1,5 @@
-<svg width="20" height="20" version="1.0" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
- <rect transform="scale(1 -1)" y="-20" width="20" height="20" rx="1.25" ry="1.25" fill="#003f87"/>
+<svg width="20" height="20" version="1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns="http://www.w3.org/2000/svg">
+ <rect transform="scale(1 -1)" y="-20" width="20" height="20" rx="2" ry="2" fill="#003f87"/>
  <use transform="matrix(.92938 0 0 -.92938 19977 4810.8)" width="300" height="240" fill="#fff" stroke="#000" stroke-linejoin="round" stroke-width="309" xlink:href="#state_outline"/>
- <path fill="#fff" d="M1 1H19V15H1z"/>
+ <rect style="fill:#fff;fill-rule:evenodd;stroke:none" width="18" height="14" x="1" y="1" rx="1" ry="1"/>
```

Comment:
Should the white portion of the shield have rounded bottom corners? It looks more approachable, but it’s inconsistent with the state highway shields of Colorado and Minnesota. (Making the bottom strip of Colorado’s flag concave would be slightly weird, but only when squinting.)

Reply:
Personally, I think it looks cute. But they're probably fine either way. These particular angles aren't in the very corner of the shield, so this choice wasn't in the _word_ of the guidelines, but in the spirit of the guidelines.

![Screenshot from 2022-05-10 20-56-59](https://user-images.githubusercontent.com/1732117/167748114-c0aa9646-5199-4671-9e27-24f41021195a.png) ![Screenshot from 2022-05-10 20-57-12](https://user-images.githubusercontent.com/1732117/167748115-6836b803-8cc0-47b0-aa56-68c827bed9fc.png)
![Screenshot from 2022-05-10 20-58-05](https://user-images.githubusercontent.com/1732117/167748116-be64604e-c8be-47f4-b8b4-fb02968f1042.png) ![Screenshot from 2022-05-10 20-58-21](https://user-images.githubusercontent.com/1732117/167748117-1dde7b63-bc45-4258-ad95-900862bbadda.png)

Side note, I think I want to put the Minnesota outline on the state shield. Having that might help to disambiguate from Intercounty routes once those get mapped.

Reply:
Here's the state outline:
![Screenshot from 2022-05-10 21-41-46](https://user-images.githubusercontent.com/1732117/167751899-bd40d656-8ebd-49eb-bbf1-b0a04960d1cd.png) ![Screenshot from 2022-05-10 21-42-06](https://user-images.githubusercontent.com/1732117/167751900-48c9e272-35d1-45d3-ab91-d28acd262fc7.png)